scrapers/GroobyClub.yml

```diff
@@ -0,0 +1,43 @@
+name: "GroobyClub"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      # checked for all sites https://www.grooby.club/tour/
+      - asianamericantgirls.com
+      - canada-tgirl.com
+      - euro-tgirls.com
+      - grooby.club
+      - hazel-tucker.com
+      - krissy4u.com
+      - russian-tgirls.com
+      - tgirl40.com
+      - tgirlbbw.com
+      - tgirlpornstar.com
+      - tgirlpostop.com
+      - transexdomination.com
+      - ts-castingcouch.com
+      - uk-tgirls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="trailer_videoinfo"]//h3/text()
+      Date:
+        selector: //div[@class="trailer_videoinfo"]//b[contains(.,"Added")]/following-sibling::text()[1]
+        postProcess:
+          - replace:
+              - regex: ^-
+                with: ""
+          - parseDate: Jan 2, 2006
+      Details: //div[@class="trailer_videoinfo"]/p[4]/text()
```

Comment:
Does not work for all sites as
- not all sites have 4 paragraphs
- some sites have extra `<l>`'s before the `<p>`'s
```yaml
      Details: //div[@class="trailer_videoinfo"]//p[not(b)]
```
may work better, not tested on all sites though